content/integrations/AWS-Integration/AWS-IAM-Installation.md

```diff
@@ -125,58 +125,40 @@ If you want to use a limited read only access policy, you’ll need to create a
 
 ```json
 {
-"Version": "2012-10-17",
-"Statement": [
-  {
-    "Action": [
-      "autoscaling:Describe*",        
-      "ce:Get*",
-      "cloudwatch:Describe*",
-      "cloudwatch:Get*",
-      "cloudwatch:List*",
-      "dynamodb:Describe*",
-      "dynamodb:Get*",
-      "dynamodb:List*",
-      "ec2:Describe*",
-      "ec2:GetConsoleOutput",
-      "ecs:Describe*",
-      "ecs:List*",
-      "elasticache:Describe*",
-      "elasticache:List*",
-      "elasticloadbalancing:Describe*",
-      "elasticmapreduce:Describe*",
-      "elasticmapreduce:List*",
-      "iam:Get*",
-      "kinesis:DescribeStream",
-      "kinesis:Get*",
-      "kinesis:List*",
-      "lambda:List*",
-      "rds:Describe*",
-      "rds:ListTagsForResource",
-      "redshift:Describe*",
-      "mq:List*",
-      "mq:Describe*",
-      "s3:Describe*",
-      "s3:Get*",
-      "s3:List*",
-      "sqs:Get*",
-      "sqs:List*",
-      "tag:Get*"
-    ],
-    "Effect": "Allow",
-    "Resource": "*"
-  }
-]
+  "Version": "2012-10-17",
```

Comment:
Wasn't this policy for monitoring?

Reply:
I'm unsure. That policy also contains s3 permissions, but more broadly right?

Should i keep the old section and clarify them as two separate minimum perm policies? one based in cost and the other monitoring?

Reply:
I'm not sure either to be honest. @rclagett what do you think?

Reply:
It is for monitoring, which we still do. So the policy should be kept. If we want a third policy  - minimum for cost performance  metrics that would be EC2, EBS, RDS, S3, ELB, ASG - then we could do that.

The three would be minimal for monitoring, minimal for cost performance metrics, minimal for billing files.